aws/components/s3/setup.ftl

```diff
@@ -19,7 +19,7 @@
 
     [#local roleId = resources["role"].Id ]
 
-    [#local versioningEnabled = solution.Lifecycle.Versioning ]
+    [#local versioningEnabled = solution.Versioning!solution.Lifecycle.Versioning ]
```

Comment:
Lets add a @warn here with the deprecation details if its found in the lifecycle section as configured.

Reply:
Done